src/core/agent/records/credentialMetadataRecord.types.ts

```diff
@@ -5,7 +5,7 @@ interface CredentialMetadataRecordProps {
   id: string;
   createdAt?: Date;
   isArchived?: boolean;
-  isDeleted?: boolean;
+  pendingDeletion?: boolean;
```

Comment:
I don't think we need this in the props. We can just set it to `false` immediately.